nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/rest/RestESDocumentsTest.java

```diff
@@ -128,7 +128,7 @@ public void iCanPerformESQLPageProviderOnRepository() throws IOException, Interr
 
         JsonNode node = mapper.readTree(response.getEntityInputStream());
         // Verify results
-        assertEquals(19, getLogEntries(node).size());
+        assertEquals(20, getLogEntries(node).size());
```

Comment:
Indeed, previously we get a result node with 19 items, but page size is 20 and the total size of query was 21. I didn't succeed to reproduce it after the fix?